pkg/api/types.go

```diff
@@ -639,9 +645,13 @@ type Container struct {
 	Name string `json:"name"`
 	// Required.
 	Image string `json:"image"`
-	// Optional: The docker image's entrypoint is used if this is not provided; cannot be updated.
+	// Optional: The docker image's entrypoint is used if this is not provided;
+	// cannot be updated; variable references ("$(VAR_NAME)") are expanded according
```

Comment:
You could also refer the reader to the rules for how EnvVar.Value is expanded rather than to the full spec